src/parse_args.ts

```diff
@@ -58,18 +56,11 @@ program
   .option('--sandbox', 'enable chromium sandboxing')
   .option('--rich-events', 'Mimics a heartbeat run')
   .option(
-    '--ws-endpoint <endpoint>',
-    'Browser WebSocket endpoint to connect to'
-  )
-  .option(
-    '--pause-on-error',
-    'pause on error until a keypress is made in the console. Useful during development'
+    '--capabilities <features...>',
```

Comment:
One small change, we should name this `--capability` I think, because as a plural it sounds like it maybe takes a comma separated list rather than multiple uses

Reply:
Good call